src/modules/Order/Controller/Admin.php

```diff
@@ -85,8 +85,9 @@ public function get_index(\Box_App $app)
     public function get_new(\Box_App $app)
     {
         $api = $this->di['api_admin'];
-        $product = $api->product_get(['id' => $this->di['request']->getPost('product_id')]);
-        $client = $api->client_get(['id' => $this->di['request']->getPost('client_id')]);
+
+        $product = $api->product_get(['id' => isset($_POST['product_id']) ? $_POST['product_id'] : null]);
```

Comment:
Can we do do `$_POST['product_id'] ?? null` instead?
Same for line under this

Reply:
Need, we can. 🙂